README.md

```diff
@@ -9,12 +9,53 @@ If you prefer Kubernetes deployment, there is a sample Kubernetes configuration
 
 ## Installing `nginx` and `docker`
 
-First, let's install `nginx` and `docker` using the commands below if you have not already installed them on your machine.
+First, update repositories cache:
 
 ```bash
 sudo apt update
+```
+
+Now, install `nginx`:
+
+```bash
 sudo apt install nginx
-sudo snap install docker
+```
+
+Then install docker using the [recommended method](https://docs.docker.com/engine/install/ubuntu/) from docker website:
```

Comment:
Can we have a link to the docker's official installation instructions instead of having copy of it here?